From 0b6b845a79eefc1d0cf7ced4a1a66ada96abef16 Mon Sep 17 00:00:00 2001
From: Jason Monk <jmonk@google.com>
Date: Wed, 23 May 2018 16:08:11 -0400
Subject: [PATCH 196/306] Follow supportlib changes in SliceManager

Test: make
Bug: 78898947
Change-Id: Ibb2bdc62798d7e0ee93dc0609a8871041abba108
(cherry picked from commit f852f56e27e768839dc4da84638bc88d21f6ca1c)
---
 .../SystemUI/src/com/android/keyguard/KeyguardSliceView.java  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/keyguard/KeyguardSliceView.java b/packages/SystemUI/src/com/android/keyguard/KeyguardSliceView.java
index 52e1c167ffc..24875d7dadf 100644
--- a/packages/SystemUI/src/com/android/keyguard/KeyguardSliceView.java
+++ b/packages/SystemUI/src/com/android/keyguard/KeyguardSliceView.java
@@ -58,7 +58,7 @@ import java.util.function.Consumer;
 
 import androidx.slice.Slice;
 import androidx.slice.SliceItem;
-import androidx.slice.SliceManager;
+import androidx.slice.SliceViewManager;
 import androidx.slice.core.SliceQuery;
 import androidx.slice.widget.ListContent;
 import androidx.slice.widget.RowContent;
@@ -373,7 +373,7 @@ public class KeyguardSliceView extends LinearLayout implements View.OnClickListe
     }
 
     public void refresh() {
-        Slice slice = SliceManager.getInstance(getContext()).bindSlice(mKeyguardSliceUri);
+        Slice slice = SliceViewManager.getInstance(getContext()).bindSlice(mKeyguardSliceUri);
         onChanged(slice);
     }
 
-- 
2.17.1

